app/views/devise/confirmations/new.html.erb

```diff
@@ -6,11 +6,11 @@
   <h3 class="mb-5">A confirmation email has been sent to <a href="mailto:<%= proper_email %>"><%= proper_email %></a></h3>
   <div class="mt-8 mb-6 crayons-card p-7 align-left mx-auto">
     <h2 class="pb-4">Didn't get the email?</h2>
-    <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+    <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }, data: { testid: "resend-confirmation-form" }) do |f| %>
```

Comment:
Adding test ID attributes are pretty standard. See [Cypress Best Practices](https://docs.cypress.io/guides/references/best-practices.html#How-It-Works) and [Making your UI tests resilient to change](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change).